src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,28 @@
+@mixin circle($size) {
+  width: $size;
+  height: $size;
+  border-radius: 50%;
+}
+
+@mixin positioning {
+  @include circle(25px);
+
+  position: absolute;
+  top: 50%;
+  left: 50%;
+
+  transform: translate(-50%, -50%);
+}
```

Comment:
It doesn't look like reusable stuff. As for me it'll be better to create a placeholder with centering functionality